aip/general/0008.md

```diff
@@ -7,22 +7,49 @@ placement:
   order: 40
 ---
 
-# AIP Style guide
+# AIP Style and Guidance
 
 AIP stands for **API Improvement Proposal**, which is a design document
-providing high-level, concise documentation for API development. The goal is for
-these documents to serve as the source of truth for API-related documentation at
-Google and the way API teams discuss and come to consensus on API guidance.
+providing high-level, concise documentation for API design and development. The
+goal is for these documents to serve as the source of truth for API-related
+documentation at Google and the way API teams discuss and come to consensus on
+API guidance.
 
 AIPs are most useful when they are clear and concise, and cover a single topic
 or inquiry well. In the same way that AIPs describe consistent patterns and
 style for use in APIs, they also _follow_ consistent patterns and style.
 
 ## Guidance
 
-AIPs **must** cover a single, discrete topic, and **should** fundamentally
-answer the question, "What do I do?" with actionable guidance. AIPs **may** also
-cover what _not_ to do, but **should not** cover _only_ anti-patterns.
+- AIPs **must** cover a single, discrete topic, and provide clear, actionable
+  guidance.
+- AIPs **must not** duplicate or contradict guidance in another AIP.
+- AIPs **may** also cover what _not_ to do, but **should not** cover _only_
+  anti-patterns.
+- If AIP guidance is conditional (e.g. a design pattern such as Jobs), the
+  guidance **must** clearly explain under what conditions the guidance should be
+  followed.
+
+AIPs **must** contain API guidance that provide a clear benefit to authors of
```

Comment:
This sentence now contains redundant text: "to authors of
one or more types of clients or their authors". Maybe just remove "or their authors".